### DISCUSSION TITLE
Bitdefender keeps reporting Trojan.GenericKD in Blazor WebAssembly (While publishing locally and loading website)

### DISCUSSION THREAD

**Discussion by chintanbug** [dotnet/runtime#67640] [General] [answered]
Hi, I am using Blazor WebAssembly with Visual Studio 2022
I am able to develop the website without any issue...
But somehow when i publish it DEBUG/RELEASE BitDefender gets the following such threats...

1) obj\Debug\net6.0\linked\Link.semaphore is infected with **Trojan.GenericKD.39437478** and was moved to quarantine.
2) _framework/System.Private.Xml.Linq.dll Threat name: **Trojan.GenericKD.39392755**

Can someone please help me?
I ran full scan of the system and found nothing, this is something i get only when i publish.

**Comment by hez2010** [comment, marked as answer]
False positives. 
I think you need to report them to BitDefender to resolve it.
